### DISCUSSION TITLE
How to custom separator like  .xl_focus_-translate-y-1\/2:focus

### DISCUSSION THREAD

**Discussion by bestony** [tailwindlabs/tailwindcss#2340] [Help]
my runtime not supporty `\` and `/`， i want to change it to `_`

## What I have try?

I try to change separator in config，but not effect it.

 https://tailwindcss.com/docs/configuration#separator

## Whic Runtim i use ?

WeChat MiniProgram [You can find docs here](https://developers.weixin.qq.com/miniprogram/en/dev/framework/view/wxss.html)

## What I want ?
1. A configuration that i can change `\` and `/`.
2. A tutorial for how to adapt wechat miniprogram

**Comment by adamwathan** [comment]
In this case you just need to change the values in your config:

```diff
  module.exports = {
    theme: {
      translate: (theme, { negative }) => ({
        ...theme('spacing'),
        ...negative(theme('spacing')),
        '-full': '-100%',
-       '-1/2': '-50%',
+       '-1_2': '-50%',
-       '1/2': '50%',
+       '1_2': '50%',
        full: '100%',
      })
    }
  }
```

I would recommend using `tailwindcss init --full` to scaffold the full config file and replacing all "/" with another character.

**Reply by bestony** [reply]
i will try it now,and will post result here.
